training/src/app/chuck-norris-main/chuck-norris-main.component.css

```diff
@@ -0,0 +1,117 @@
+
+.top-bar > #top-bar-img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 80%;
+    height: 60%;
+}
+
+.buttons {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.buttons > #toggle-buttons {
+    border-radius: 4px 4px;
+    margin-top: 3%;
+    margin-bottom: -5%;
+    height: 30px;
+    border: 1px solid #F05924;
```

Comment:
lets move all colors in this file to styles.scss as a const
$orange-color: #F05924;
then u can use here and other places $orange-color
thats way u wont have duplication, and if u need to change - its 1 place to change